test/test_model_emulation.jl

```diff
@@ -478,3 +478,55 @@ end
         GLPK_optimizer,
     )
 end
+
+@testset "Test serialization of InitialConditionsData" begin
+    template = get_thermal_standard_uc_template()
+    sys = PSB.build_system(
+        PSITestSystems,
+        "c_sys5_pglib";
+        add_single_time_series = true,
+        force_build = true,
+    )
+    optimizer = Cbc_optimizer
```

Comment:
same here, it is better to use the predefined optimizers so we can change them in a single place.

Reply:
Maybe I'm missing you, but that is exactly what I tried to do here. Every `run` in this test is controlled by a single optimizer. You someone wants to use GLPK_optimizer instead of Cbc_optimizer, they can change it on this line.